data/distro.yml

```diff
@@ -795,7 +795,6 @@
       </li>
     </ol>
     
```

Comment:
Please, also remove the four spaces ("    ") and replace them with just an empty line as I mentioned in my previous comment. Thanks!